cunyfirstapi/tests/tests.py

```diff
@@ -10,11 +10,24 @@
 '''
 ###***********************************###
 import unittest
+from ..locations_enum import Location
+from ..cunyfirstapi import CUNYFirstAPI
 
 class TestTest(unittest.TestCase):
     def test(self):
         self.assertEqual(True, True)
 
+
+class ClassSearchTest(unittest.TestCase):
```

Comment:
This isnt gonna work. 

1. Add the argsParse option to main, store a global var for username and password and use those gl